manifests/j/JetBrains/IntelliJIDEA/Community/EAP/222.2270.31/JetBrains.IntelliJIDEA.Community.EAP.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: JetBrains.IntelliJIDEA.Community.EAP
+PackageVersion: 222.2270.31
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download-cdn.jetbrains.com/idea/ideaIC-222.2270.31.exe
+  InstallerSha256: DF8C6C67FD32474B8963C95D269BFDE1F7571AF08005E12290142E8968F25679
+  ProductCode: IntelliJ IDEA Community Edition 221.5787.3
```

```suggestion
  ProductCode: IntelliJ IDEA Community Edition 222.2270.31
```

Can your script do a find and replace?